### ISSUE TITLE
Slow speeding up replays with speed higher than x10

### ISSUE THREAD

**Issue by TehPlayer14** [Kromster80/kam_remake#42]
The technology has already made fasters processor than they were before

and KaM Remake doesn't realy use extra cores/processing power on replay speed up on x300 makes game output no frames at all when using it but today cpu's can atleast handle at least 1 frame with x300 speed

Like speeding up for 1 sec should bring you 300s ingame time it does only bring you back half of it so 150s or less

**Comment by Kromster80** [comment]
Multithreading of games logic code is an enormous amount of work and redesign of the bigger half of games engine. It is impractical to do.

Optimizing the game for speed is more viable solution. However we have already applied many simple and average optimizations.

**Comment by TehPlayer14** [comment]
Would vulcan change something?

**Comment by Kromster80** [comment]
Not really. It is just another rendering API, much like OpenGL we already use.
